packages/api/src/mocks.ts

```diff
@@ -210,13 +226,18 @@ export function clearMocks(): void {
     return
   }
 
-  if (window.__TAURI_INTERNALS__?.convertFileSrc)
-    // @ts-expect-error "The operand of a 'delete' operator must be optional' does not matter in this case
-    delete window.__TAURI_INTERNALS__.convertFileSrc
-  if (window.__TAURI_INTERNALS__?.invoke)
-    // @ts-expect-error "The operand of a 'delete' operator must be optional' does not matter in this case
-    delete window.__TAURI_INTERNALS__.invoke
-  if (window.__TAURI_INTERNALS__?.metadata)
-    // @ts-expect-error "The operand of a 'delete' operator must be optional' does not matter in this case
-    delete window.__TAURI_INTERNALS__.metadata
+  // @ts-expect-error "The operand of a 'delete' operator must be optional' does not matter in this case
```

Comment:
Nitpick, but the double quote is closed incorrectly
```suggestion
  // @ts-expect-error "The operand of a 'delete' operator must be optional" does not matter in this case
```